Content.IntegrationTests/Tests/Minds/MindTests.ReconnectTests.cs

```diff
@@ -134,11 +146,14 @@ public async Task TestVisitingReconnect()
         await DisconnectReconnect(pair);
 
         // Player is back in control of the visited mob, mind was preserved
-        Assert.That(mind == GetMind(pair));
-        Assert.That(!entMan.Deleted(original));
-        Assert.That(!entMan.Deleted(visiting));
-        Assert.That(mind.CurrentEntity == visiting);
-        Assert.That(mind.CurrentEntity == visiting);
+        Assert.Multiple(() =>
+        {
+            Assert.That(GetMind(pair), Is.EqualTo(mind));
+            Assert.That(!entMan.Deleted(original), Is.False);
+            Assert.That(!entMan.Deleted(visiting), Is.False);
```

Comment:
Both of the checks here shouldn't have the !s. That's what the Is.False s are for.